docs/recipes/github-actions.md

```diff
@@ -0,0 +1,96 @@
+# Using semantic-release with [GitHub Actions](https://help.github.com/en/categories/automating-your-workflow-with-github-actions)
+
+## Environment variables
+
+The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured with [Secret variables](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
+
+For this example, you'll need to setup a `NPM_TOKEN` and a `GH_TOKEN`, this last one is recommended to use [`@semantic-release/git`](https://github.com/semantic-release/git) with a **CI user who had permission to push to master branch** to update `package.json` file.
```

Comment:
The `GH_TOKEN` is only needed if you use the `git` plugin and have branch protection enabled as far as I know. I use semantic-release with just `NPM_TOKEN` and the provided `GITHUB_TOKEN`, and we should probably document that as the standard case as it's easier to setup?

Reply:
Hum, I changed a little bit the documentation. Please, take a look again @gr2m .
Thanks!